README.md

```diff
@@ -53,9 +53,18 @@ or browse directly the [docs folder](./docs).
 
 #### Node Version Requirement
 
-Building deck.gl from source has a dependency on node `0.12` or higher.
+Building deck.gl from source has a dependency on node `4.0` or higher.
 Either upgrade to a newest version, or install something like [nvm](https://github.com/creationix/nvm).
 
+#### Install yarn
+deck.gl uses [yarn] (https://www.npmjs.com/package/yarn) to manage packages.
+To develop deck.gl, install yarn with brew
```

Comment:
This is mac specific. To avoid confusing external users, maybe best to also provide a link https://yarnpkg.com/en/docs/install